elevation_mapping/include/elevation_mapping/ElevationMapping.hpp

```diff
@@ -175,6 +183,18 @@ class ElevationMapping
    */
   void stopMapUpdateTimer();
 
+  /*!
+   * Get a gridMapMessage of a Submap or rawSubmap
+   * @param position requested position of the submap.
+   * @param length requested length of the submap.
+   * @param layers requested layers of the submap.
+   * @param useRaw if true, returns submap of rawMap, if false, returns submap of fusedMap.
+   * @param isSuccess set true if successful.
+   * @return submap message.
+   */
+  grid_map_msgs::GridMap getSubmapMessage(grid_map::Position position, grid_map::Length length,
```

Comment:
Input parameters should be const ref (except for primitives), could even the method be const?